src/transformers/models/auto/tokenization_auto.py

```diff
@@ -298,6 +298,7 @@
             ("roc_bert", ("RoCBertTokenizer", None)),
             ("roformer", ("RoFormerTokenizer", "RoFormerTokenizerFast" if is_tokenizers_available() else None)),
             ("rwkv", (None, "GPTNeoXTokenizerFast" if is_tokenizers_available() else None)),
+            ("scibart", ("SciBartTokenizer", None)),
```

Comment:
Wondering if we should protect the import with `is_sentencepiece_available()` I am unsure here since it seems that it is now a dependency on `transformers`: https://github.com/huggingface/transformers/blob/a8732e09bb230848f1ab0f3cab9e533b412ca143/setup.py#L166 / I think this is fine but want to double check just in case cc @amyeroberts

Reply:
No, this should also be protected with the `is_sentencepiece_available()` check here, as it's imported in `tokenization_scibart.py`. Sentencepiece isn't install if a user runs `pip install transformers`.

Reply:
The tokenizer should also safely imported in the model init with a `is_sentencepiece_available` check e.g. like [here for albert](https://github.com/huggingface/transformers/blob/aea7b23b57ea0bbe15c42114d35b3c7a4d79d7eb/src/transformers/models/albert/__init__.py#L33)

Reply:
Thanks for confirming @amyeroberts !